source/elements/oneTBB/source/containers/concurrent_hash_map_cls/iterators.rst

```diff
@@ -52,3 +52,21 @@ equal_range
     a pair of iterators ``{f, l}``, where ``f`` is an iterator to this element,
```

Comment:
```suggestion
    returns a pair of iterators ``{f, l}``, where ``f`` is an iterator to this element,
```

Reply:
I have rephrased this section in the other manner.